Documentation/docs-mobile/messages/adb0010.md

```diff
@@ -25,6 +25,6 @@ errors related to APK installation. An error message of the form
 
 ## Implementation notes
 
-Note that nothing in the open source xamarin-android repository
-emits ADB0010, as features such as debugging and "fast deployment"
-are implemented in the proprietary Xamarin.Android additions.
+Note that nothing in the open source <https://github.com/xamarin/xamarin-android>
```

Comment:
Is this supposed to be a markdown link? Or are we really wanting the text `<https://github.com/xamarin/xamarin-android>` to show?

Reply:
This is in here a couple other times.

Reply:
Yes, that's supposed to be a Markdown link; see rendered: https://learn.microsoft.com/en-us/dotnet/android/messages/adb0010

> Note that nothing in the open source <https://github.com/xamarin/xamarin-android> repository emits ADB0010, …

I think there are two implied questions here:

 1. Should we be mentioning it *here*?
 2. Should we even be mentioning xamarin/xamarin-android, anywhere?

Regarding (1), perhaps not: it's likely extraneous information that doesn't really help anybody, and thus we should consider removing the `## Implementation notes` sections from *all* the `adb*` messages.

Regarding (2), it *is* the current repo location (though we want to do that around June…), and it does make sense to mention it when relevant, e.g. in `xa0107.md`:

> ```markdown
> Consider submitting a [bug](https://github.com/xamarin/xamarin-android/wiki/Submitting-Bugs,-Feature-Requests,-and-Pull-Requests)
> if you are getting this warning under normal circumstances.                                                                  
> ```

We could consider creating an `aka.ms` shortlink instead of directly linking to the repo.

That, there are also the Release notes, which directly link to xamarin/xamarin-android and are directly mentioned in `TOC.yml` (if not yet rendered), and we likely *don't* want to create short links for each one of those…